include/pctDDParkerShortScanImageFilter.hxx

```diff
@@ -0,0 +1,150 @@
+#ifndef __pctDDParkerShortScanImageFilter_txx
```

Comment:
_txx -> _hxx in all files. I'm atually surprised this is not checked by kwstyle or clang-format